Source/Charts/Charts/RadarChartView.swift

```diff
@@ -118,19 +118,19 @@ open class RadarChartView: PieRadarChartViewBase
         
         if drawWeb
         {
-            renderer!.drawExtras(context: context)
+            renderer?.drawExtras(context: context)
```

Comment:
`guard let data = _data, let renderer = renderer else { return }` at the beginning of the function,

Reply:
done